spec/system/viewing_sidebar_mobile_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+describe "Viewing sidebar mobile", type: :system, js: true, mobile: true do
+  fab!(:user) { Fabricate(:user) }
+  let(:sidebar_dropdown) { PageObjects::Components::SidebarHeaderDropdown.new }
+  let(:composer) { PageObjects::Components::Composer.new }
+
+  before do
+    @original_disable_animation = Capybara.disable_animation
+    Capybara.disable_animation = true
```

Comment:
Disabling animation here because dealing with a sliding in menu makes the tests extremely brittle.

Reply:
Nit: can also consider using `#around` to avoid splitting the logic between `#before` and `#after` blocks.

```ruby
  around do |example|
    original_disable_animation = Capybara.disable_animation
    Capybara.disable_animation = true

    example.run

    Capybara.disable_animation = original_disable_animation
  end
```

Reply:
O nice didn't know about this :+1: